pwnlib/update.py

```diff
@@ -191,10 +191,13 @@ def perform_check(prerelease=current_version.is_prerelease):
     return command
 
 def check_automatically():
+    xdg_config_home = os.environ.get('XDG_CONFIG_HOME') or "~/.config"
+
     if should_check():
         message  = ["Checking for new versions of %s" % package_name]
         message += ["To disable this functionality, set the contents of %s to 'never' (old way)." % cache_file()]
-        message += ["""Or add the following lines to ~/.pwn.conf (or /etc/pwn.conf system-wide):
+        message += ["Or add the following lines to %s (or /etc/pwn.conf system-wide):" % (xdg_config_home + "/pwn.conf")]
```

Comment:
List both locations, not one or the other.

Reply:
Also we need to handle the case where XDG_CONFIG_HOME is not set (e.g. macOS) and not recommend that path.

Reply:
Are you referring to `~/.config` vs `$XDG_CONFIG_HOME` discussion above, or you're suggesting that all three of `/etc/pwn.conf`, `~/.config/pwn.conf` and `~/.pwn.conf` should be mentioned here?

Reply:
> all three of /etc/pwn.conf, ~/.config/pwn.conf and ~/.pwn.conf should be mentioned here

This, we should continue to support loading from ~/.pwn.conf even if $XDG_HOME/pwn.conf exists.  If they both exist, load them both, don't really care about ordering.

Reply:
Probably ordering should be

/etc/pwn.conf
XDG_CONFIG_HOME/pwn.conf
~/.pwn.conf

This keeps the current behavior (~/.pwn.conf wins), while adding support for the XDG_CONFIG_HOME you're proposing.

Reply:
I suppose the macOS question was answered in the duscussion above?

Reply:
Yeah, if `$XDG_CONFIG_HOME` is not set, then just don't try to load that file.